src/core/variable.jl

```diff
@@ -342,11 +344,35 @@ end
 "variable: `0 <= branch_ne[l] <= 1` for `l` in `branch`es"
 function variable_branch_ne(pm::GenericPowerModel, n::Int=pm.cnw)
     branches = pm.ref[:nw][n][:ne_branch]
-    pm.var[:nw][n][:branch_ne] = @variable(pm.model, 
+    pm.var[:nw][n][:branch_ne] = @variable(pm.model,
         [l in keys(branches)], basename="$(n)_branch_ne",
-        lowerbound = 0, 
+        lowerbound = 0,
         upperbound = 1,
         category = :Int,
         start = getstart(branches, l, "branch_tnep_start", 1.0)
     )
 end
+
+
+"variable: `0 <= i[l] <= (Imax)^2` for `l` in `branch`es"
+function variable_series_current_magnitude_sqr(pm::GenericPowerModel, n::Int=pm.cnw; bounded = true)

```

Comment:
I don't think you need this constraint.  You can move this implementation into this shared variable space,

https://github.com/lanl-ansi/PowerModels.jl/blob/master/src/form/wr.jl#L399